core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -168,7 +168,7 @@ class BrokerToControllerChannelManagerImpl(
   threadNamePrefix: Option[String],
   retryTimeoutMs: Long
 ) extends BrokerToControllerChannelManager with Logging {
-  private val logContext = new LogContext(s"[broker-${config.brokerId}-to-controller] ")
+  private val logContext = new LogContext(s"[BrokerToControllerChannelManager broker=${config.brokerId} name=$channelName] ")
```

Comment:
Shall we use this in defining the thread name passed to `BrokerToControllerRequestThread` as well? The thread name is used for the `logIdent` in `ShutdownableThread`.